requirements-test.txt

```diff
@@ -1,2 +1,3 @@
 pytest==2.9.2
 py==1.4.31
+pygments==2.1.3
```

Comment:
Looks like I'm running into this as well: https://travis-ci.org/awslabs/chalice/builds/148965791

Let's go ahead and keep this in here.